03/Dog.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include "Animal.h"
+
+class Dog : public Animal
+{
+public:
+	Dog();
+	Dog(int age, const char* size);
+	~Dog();
+
+	char* GetSize() const;
+
+private:
+	char* mSize;
```

Comment:
dog has size . .cat has name... doesn' tmake huge sense...

so this example doesn't show proper OOP.... 

maybe we can find functions that's unique to cat .. and another functions taht's unique to dog?